docs.wrm/api/providers/types.wrm

```diff
@@ -285,7 +285,7 @@ transaction was mined.
 
 _property: transaction.raw => string<[[DataHexString]]>
 The serialized transaction. This may be null as some backends do not
-rpopulate it. If this is required, it can be computed from a **TransactionResponse**
+repopulate it. If this is required, it can be computed from a **TransactionResponse**
```

Comment:
typo

Reply:
Thanks! It was meant to be populate. :)

Fixing it now, as I'm doing a large sweep of the docs issues. :)